backerymails.post_update.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for Backerymails module.
+ */
+
+use Drupal\Core\Database\Database;
+
+/**
+ * Migrate data from the old existing Backerymails table to the new one.
+ *
+ * @see backerymails_update_8001()
+ */
+function backerymails_post_update_8001_migrate_data(&$sandbox = NULL) {
+  $database = Database::getConnection();
+
+  if ($database->schema()->tableExists('backerymails_sended_mail') && $database->schema()->tableExists('backerymails_sent_mails')) {
```

Comment:
use early return that explain:

as post_update will always run independent of hook_update, but only once. Ensure we don't run migrate if the module was already installed postschema 8001 (we may tests that instead of tableexists)